feature-libs/product-configurator/rulebased/components/attribute/quantity/configurator-attribute-quantity.component.spec.ts

```diff
@@ -64,7 +79,27 @@ describe(' ConfiguratorAttributeQuantityComponent', () => {
 
   it('should call handleQuantity on event onChangeQuantity', () => {
     component.onChangeQuantity();
+    expect(component.changeQuantity.emit).toHaveBeenCalled();
+  });
 
+  it('should not emit change event on quantity change if debounce time has not yet passed', fakeAsync(() => {
+    component.quantity.setValue(changedQty);
+    fixture.detectChanges();
+    tick(fakeDebounceTime - 100);
+    expect(component.changeQuantity.emit).toHaveBeenCalledTimes(0);
```

Comment:
instead of assert zero times, you can also use 'not' as below. I think it's a bit easier to read.
`expect(component.changeQuantity.emit).not.toHaveBeenCalled();`